script.js

```diff
@@ -11,4 +11,137 @@ document.querySelectorAll(".nav-link").forEach(n => n.
+    const elements = document.createElement('div')
+    const button = document.createElement("button")
+    img.src = `${projects.img1}`
+    article.classList.add('card1')
+    button.classList.add('btn')
+    elements.classList.add('elements1')
+    button.innerText = 'See Project'
+
+    elements.innerHTML += `
+    <h2 class="title">${projects.title1}</h2>
+                        <ul class="highlights">
+                            <li>CANOPY</li>
+                            <li><img class="counter" src="Counter.png" alt="counter"></li>
+                            <li>Back End Dev</li>
+                            <li><img class="counter" src="Counter.png" alt="counter"></li>
+                            <li>2015</li>
+                        </ul>
+                        <p class="description">${projects.description}</p>
+                        <ul class="languages">
+                            <li class="stack">${projects.technologies[0]}</li>
+                            <li class="stack">${projects.technologies[1]}</li>
+                            <li class="stack">${projects.technologies[2]}</li>
+                        </ul>
+    `
+    elements.append(button)
+    divImg.append(img)
+    article.append(divImg, elements)
+    section.append(article)
+
+    button.addEventListener("click", () => {
+        popup.style.display = 'flex'
+        section.style.filter = 'blur(5px)'
+        document.body.style.overflow = 'hidden'
+        popup.innerHTML = `
+    <div class="popup-header">
+                    <h2 class="popup-title">${projects.title1}</h2>
+                    <div class="cancel-icon"><img src="./ICONS/cancel-icon.png"></div>
+                </div>
+                <ul class="popup-highlights">
+                    <li>CANOPY</li>
+                    <li><img class="counter" src="Counter.png"></li>
+                    <li>Back End Dev</li>
+                    <li><img class="counter" src="Counter.png"></li>
+                    <li>2015</li>
+                </ul>
+                <img src="${projects.img2}">
+                <div class="popup-content">
+                    <div class="left-section">
+                        <p class="project-descritpion">${projects.description2}</p>
+                    </div>
+                    <div class="right-section">
+                        <ul class="stacks popup">
+                            <li class="stack">Html</li>
+                            <li class="stack">Css</li>
+                            <li class="stack">Javascript</li>
+                        </ul>
+                        <div class="popup-btns">
+                            <a href="${projects.liveDemo}" class="popup-btn" target="_blank">See live <img src="./ICONS/live.png"></a>
+                            <a href="#" class="popup-btn" target="_blank">See source<img src="./ICONS/github.png"></a>
+                        </div>
+                    </div>
+                </div>
```

Comment:
- [ ] Kindly fix the display on the "See live" and "See source" buttons to match the Figma design **on both the desktop and mobile view** (the icon is displayed next to the text, both buttons are displayed side by side not under one another).
- [ ] Kindly add a line between the technology tags and the buttons to match the Figma design.

| | Your design | Figma design |
| --- | --- | --- |
| Desktop View | ![image](https://user-images.githubusercontent.com/62290545/195357257-7eebbef0-6623-43b8-9e14-545eee624b02.png) | ![image](https://user-images.githubusercontent.com/62290545/195357329-e07b4a63-c311-4d74-ab95-00a29b93332c.png) |
| Mobile View | ![image](https://user-images.githubusercontent.com/62290545/195359545-326e6b5d-0ddf-4506-a2ec-1f3a0718d086.png) | ![image](https://user-images.githubusercontent.com/62290545/195359693-d3b2ba07-af1a-48f3-a610-bf24316cfe88.png) |